.github/workflows/golangci-lint.yml

```diff
@@ -14,11 +14,11 @@ jobs:
     steps:
       - uses: actions/setup-go@v3
         with:
-          go-version: ^1.14
+          go-version: ^1.17
       - uses: actions/checkout@v2
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
         with:
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
-          version: v1.46
+          version: v1.46.2
```

Comment:
The comment above said that this "must be specified without patch version". The plugin will automatically use the latest patch version.